doc-src/dev.rst

```diff
@@ -0,0 +1,369 @@
+  PRs and their own list of review requests, and also to provide a way to
+  request reviews from the project lead without also requesting merge.
+
+* Closing issues: We close issues when we've taken the requested action,
+  decided not to take action, resolved the question, or actively determined an
+  issue is obsolete. It is OK for "stale" issues to sit around indefinitely
+  awaiting this. Unlike many projects, we do not automatically close issues
+  just because they're old.
+
+* Stale PRs, on the other hand, are to be avoided due to bit rot. We try to
+  either merge or reject PRs in a timely manner.
+
+* Closed issues can be re-opened if new information arises, for example a
+  :code:`worksforme` issue with new reproduction steps. Please comment to ask
+  for re-opening rather than doing it yourself.
+
+GitHub issue and PR tags
+------------------------
+
+What kind of issue is it?
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:code:`bug`
+  Problem of some kind that needs to be fixed; i.e., something doesn't work.
+  This includes usability and documentation problems.
+
+:code:`enhancement`
+  Things work, but it would be better if something was different. For example,
+  a new feature proposal or refactoring.
+
+:code:`help wanted`
+  The core team does not plan to address this issue, perhaps because we don't
+  know how, but we think it would be good to address it. We hope someone from
+  the community will volunteer.
+
+:code:`key issue`
+  A particularly important or notable issue.
+
+:code:`question`
+  Support request that does not report a problem or ask for a change.
+
+What do we plan to do about it?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For all of these, leave other tags in place, e.g. :code:`bug`.
+
+:code:`deferred`
+  We do not plan to do this, or accept PRs for it, but it's not completely
+  rejected. These issues stay open, because we do not consider the deferred
+  state resolved. If you do want to submit a PR, we welcome argument to remove
+  the tag; a successful argument is almost certainly a pre-requisite for an
+  accepted PR.
+
+:code:`duplicate`
+  Same as some other previously reported issue. In addition to this tag,
+  duplicates should refer to the other issue and be closed.
+
+:code:`obsolete`
+  No longer relevant, moot, etc. Close.
+
+:code:`erroneous`
+  Not a Charliecloud issue; close. *Use caution when blaming a problem on user
+  error. Often (or usually) there is a documentation or usability bug that
+  caused the "user error".*
+
+:code:`ready to merge`
+  PRs only. Adding this tag states that the PR is complete and requests it be
+  merged to master. If the project lead requests changes, they'll remove the
+  tag. Re-add it when you're ready to try again. Lead removes tag after
+  merging.
+
+:code:`wontfix`
+  We are not going to do this, and we won't merge PRs. Close issue after
+  tagging, though sometimes you'll want to leave a few days to allow for
+  further discussion to catch mistaken tags.
+
+:code:`worksforme`
```

Comment:
Perhaps _unreproducible_ is more descriptive?

Reply:
I prefer "works for me" because it's a more specific statement: *I* can't reproduce it rather than *no one* can reproduce it.

The origins of that specific word is Bugzilla, I believe.